record-and-playback/core/scripts/post_events/post_events_analytics_callback.rb

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/ruby
+# encoding: UTF-8
+
+#
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/
+#
+# Copyright (c) 2020 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 3.0 of the License, or (at your option)
+# any later version.
+#
+# BigBlueButton is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+#
+
+require '../../core/lib/recordandplayback'
+require 'bbbevents'
+require "java_properties"
+require "jwt"
+require 'net/http'
+require 'optparse'
+require 'rubygems'
+require 'socket'
+require 'yaml'
+
+require File.expand_path('../../../lib/recordandplayback', __FILE__)
+
+file = File.open('/var/log/bigbluebutton/post_events.log', File::WRONLY | File::APPEND | File::CREAT)
+logger = Logger.new(file, 'weekly' )
+logger.level = Logger::INFO
+BigBlueButton.logger = logger
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = 'Usage: ruby post_events/post_events.rb -m <meeting_id>'
```

Comment:
Just a quick note that the script name is different. Also, I'm interested to know why to use optparse while we already we optimist (former trollop) in the other scripts.

Reply:
Good question. Why @kepstin @ritzalam ? Should I change it to optimist?

Also, I noticed that we are still using trollop in all of the other scripts. I was planning on opening a PR for that, but not sure what is the plan there